src/index.hbs

```diff
@@ -100,21 +100,21 @@ description: Better grades, together. Gradebook is the only student-built, stude
 				<div class="feature-extended-body is-revealing">
 					<h3 class="mt-0">Experiment with ease</h3>
 					<p class="mb-0">
-						See how earning different grades across assignments will affect your average
+						See how different grades will affect your average
 					</p>
 				</div>
 			</div>
 			<div class="feature-extended">
 				<div class="feature-extended-image is-revealing">
 					<img
-						src="/static/images/index/experiment.jpg"
+						src="/static/images/index/insights.jpg"
```

Comment:
Remove the old `experiment.jpg` image?

Reply:
Lol didn't see @vikaspotluri123 's comment 😄

Reply:
Nope we still need it